packages/create-redwood-app/template/api/src/lib/logger.ts

```diff
@@ -0,0 +1,88 @@
+import { createLogger, defaultLoggerOptions } from '@redwoodjs/api/logger'
```

Comment:
I take it this file isn't ready for review?

Reply:
> I take it this file isn't ready for review?

How do you mean? Sorry, I don't follow.

I used the import that I have been using in my test app having built RW and creating a `logger.ts`.

Reply:
Oh, I meant that I saw a bunch of comments and commented out code. My assumption was that it wasn't ready for review.

Reply:
@peterp I understand now.

The comments were intended to be examples:

```
/*
 * Examples of how to configure and
 * send to log transport streams
/*
```

But otherwise, the file is reviewable.

Perhaps I'll move these examples into the README. Yes, I'll do that.

There were there to also remind me when I implement setups for the transports in a future PR.